code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -381,10 +381,12 @@ GLOBAL_DATUM_INIT(canister_icon_container, /datum/canister_icons, new())
 					if(air_contents.toxins > 0)
 						message_admins("[key_name_admin(usr)] opened a canister that contains plasma in [get_area(src)]! (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)")
 						log_admin("[key_name(usr)] opened a canister that contains plasma at [get_area(src)]: [x], [y], [z]")
+						usr.create_log(MISC_LOG, "has opened a canister of plasma")
```

Comment:
```suggestion
						ui.user.create_log(MISC_LOG, "has opened a canister of plasma")
```

Reply:
its better if this is done for the whole proc, but honestly this isn't a refactor so this is good

Reply:
@Contrabang @Sirryan2002  Replaced all occurrences with `ui.user` and tested if it broke something, seemingly did not